### DISCUSSION TITLE
Email picture size issue

### DISCUSSION THREAD

**Discussion by randysrs** [YetiForceCompany/YetiForceCRM#15472] [General]
having a issue with adding Pictures to emails for workflows.

if i add a image say a 600x600 96dpi image, it seems to work . but if its a 600x700 150 or large DPI the image imports fine 
but when it saves the task type and task title is then blank . 
i can then delete the workflow task and recreate it with the smaller picture then it saves correctly and has the task type and task title.

saved with a 96dpi image
![image](https://user-images.githubusercontent.com/58384233/143400670-1c31f5d9-10a9-4c67-a99b-a8ddc385a025.png)

saved with a larger image that fails

![image](https://user-images.githubusercontent.com/58384233/143405974-7dd9d9cb-3937-4585-b21d-080bbf03a4dc.png)

**Comment by mariuszkrzaczkowski** [comment]
It looks like a bug report to me, without full information, we will not help

> > https://github.com/YetiForceCompany/YetiForceCRM/discussions/15377 < <
